src/components/backend-ai-app-launcher.ts

```diff
@@ -1182,6 +1254,23 @@ export default class BackendAiAppLauncher extends BackendAIPage {
         <div slot="content"></div>
         <div slot="footer"></div>
       </backend-ai-dialog>
+      <backend-ai-dialog id="vscode-dialog" fixed backdrop>
+        <span slot="title">${_t('session.VSCodeRemoteConnection')}</span>
+        <div slot="content" style="padding:15px 0;">
+          <div style="padding:15px 0;">${_t('session.VSCodeRemoteDescription')}</div>
+          <section class="vertical layout wrap start start-justified">
+            <h4>${_t('session.ConnectionInformation')}</h4>
+            <div><span>VSCODE Remote Password:</span>
```

Comment:
Why don't you use `VS Code` instead of `VSCODE`?
In the Visual Studio Code homepage, they use the VS Code. ([ref](https://code.visualstudio.com/))
<img width="903" alt="image" src="https://user-images.githubusercontent.com/28584164/198197968-3cfbe28b-e076-4776-87b4-02ab788907ee.png">

Reply:
Changed VSCODE to VS Code